clang/test/Driver/sycl-device-lib-bfloat16.cpp

```diff
@@ -30,6 +30,17 @@
 // RUN:   "-device pvc" %s --sysroot=%S/Inputs/SYCL -### 2>&1 \
 // RUN:   | FileCheck %s -check-prefix=BFLOAT16-NATIVE
 
+// Test that a wcl AOT compilation uses the native library.
+// RUN: %clangxx -fsycl -fsycl-targets=spir64_gen -Xsycl-target-backend \
+// RUN:   "-device wcl" %s --sysroot=%S/Inputs/SYCL -### 2>&1 \
+// RUN:   | FileCheck %s -check-prefix=BFLOAT16-NATIVE
+// RUN: %clangxx -fsycl -fsycl-targets=intel_gpu_wcl %s \
+// RUN:   --sysroot=%S/Inputs/SYCL -### 2>&1 \
+// RUN:   | FileCheck %s -check-prefix=BFLOAT16-NATIVE
+// RUN: %clang_cl -fsycl -fsycl-targets=spir64_gen -Xsycl-target-backend \
+// RUN:   "-device wcl" %s --sysroot=%S/Inputs/SYCL -### 2>&1 \
```

Comment:
```suggestion
// RUN:   "-device wcl" -- %s --sysroot=%S/Inputs/SYCL -### 2>&1 \
```
There is a separate effort to try and clean up the driver tests that are using `clang-cl` where we need to add `--` before `%s` to address potential parsing problems of the source filenames that can potentially start with `/`.  Please update all of the lines you are adding that use `clang-cl` to keep things in sync.

Reply:
Hi, @mdtoguchi 
All clang_cl cases in sycl-device-lib-bfloat16 are cleaned.